wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallGridView/CallGridViewController.swift

```diff
@@ -418,14 +419,21 @@ final class CallGridViewController: UIViewController {
               endIndex > startIndex
         else { return }
 
-        let clients = dataSource[startIndex ..< endIndex]
+        let clientsWithVideo = dataSource[startIndex ..< endIndex]
             .filter(\.isSharingVideo)
-            .map(\.streamId)
 
-        guard Set(clients) != Set(visibleClientsSharingVideo) else { return }
+        let oneStreamDisplayedExcludingSelf = clientsWithVideo.count == 1
+        let clientStreams = clientsWithVideo
+            .map { client in
+                AVSClientVideoStream(
+                    client: client.streamId,
+                    quality: oneStreamDisplayedExcludingSelf ? .high : .low
+                )
+            }
 
-        delegate?.callGridViewController(self, perform: .requestVideoStreamsForClients(clients))
-        visibleClientsSharingVideo = clients
+        guard Set(clientStreams) != Set(visibleClientsSharingVideo) else { return }
```

Comment:
suggestion: store the Set and change the comparison:
```suggestion
        guard Set(clientStreams) != visibleClientsSharingVideo else { return }
```

Reply:
done